profiles/api.py

```diff
@@ -367,7 +367,7 @@ def parse_alumni_csv(csv_path):
 
         alumni = [
             Alum(
-                learner_email=row[ALUM_LEARNER_EMAIL],
+                learner_email=str(row[ALUM_LEARNER_EMAIL]).lower(),
```

Comment:
Will this handle the case where the existing user's email is uppercase and the imported alum email is lowercase?

Reply:
@pdpinch No, because normally we only the email addresses in lowercase.

Reply:
Normally we have that format `abc@example.com`. It considered a standard format.

Reply:
@umarmughal824 what if we do a case insensitive search at https://github.com/mitodl/bootcamp-ecommerce/blob/master/profiles/api.py#L299 as well? and maybe convert email to lower while creating a user https://github.com/mitodl/bootcamp-ecommerce/blob/master/profiles/api.py#L301?

Reply:
Yeah in that case it these could be the possible combinations in my opinion like
'ABC@example.com'
'abc@EXAMPLE.COM'
'ABC@EXAMPLE.COM'